python/unityagents/grpc_communicator.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import grpc
+
+from communicator import PythonToUnityStub
+from communicator import UnityRLOutput, UnityRLInput,\
+    UnityInput, AcademyParameters, UnityInitializationInput
+from .exception import UnityTimeOutException
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("unityagents")
+
+
+class GrpcCommunicator(object):
+    def __init__(self, worker_id=0,
+                 base_port=5005):
+        """
+        Python side of the socket communication
```

Comment:
You mean "Python side of the Grpc communication"?

Reply:
Yes, Thank you